person.rb

```diff
@@ -1,26 +1,21 @@
 # person.rb
 
-#
-# Interface Class Nameable, for decorator design
-#
-class Nameable
-  def correct_name
-    raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
-  end
-end
+require_relative 'nameable'
 
 #
 # Class representing a person in the library application
 #
 class Person < Nameable
   attr_accessor :name, :age
   attr_writer :id
+  attr_reader :rentals
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     super()
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
```

Comment:
- [ ] A method for adding rentals should also be present. please see this [Link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) to the example in a previous lesson.

Reply:
changes done, thank you!